nx-dev/ui-markdoc/src/index.ts

```diff
@@ -38,6 +38,9 @@ import { tab, tabs } from './lib/tags/tabs.schema';
 import { YouTube } from './lib/tags/youtube.components';
 import { youtube } from './lib/tags/youtube.schema';
 
+// TODO fix this export
```

Comment:
Do you want to fix it before merge?

Reply:
I left this in because I used this as a component in the jsx directly for displaying the button for the GitHub release. And given we are going to work on the design, I wasn't sure whether we even end up using this one specifically or have something custom just for that changelog page.

That's why I left it in for now until we know what we're gonna use :)